test/yb_test.c

```diff
@@ -0,0 +1,351 @@
+/*
+ * Copyright (c) YugaByte, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+#include <math.h>
+#include <errno.h>
+#include <stdbool.h>
+#include <string.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/resource.h>
+#include <time.h>
+#include <unistd.h>
+
+#include <hdr/hdr_histogram.h>
+#include <hdr/hdr_histogram_log.h>
+
+#include "minunit.h"
+
+#if defined(_MSC_VER)
+#pragma warning(push)
+#pragma warning(disable: 4996)
+#endif
+
+int tests_run = 0;
+static const int YB_DEFAULT_MAX = 16777215;
+static const int YB_DEFAULT_BUCKET_FACTOR = 16;
+
+static int get_subbucket_width(hdr_histogram* h, int value)
```

Comment:
Better to give some brief comments on these util functions, esp here what is expected as the input (value), and the output.